custom/covid/management/commands/data_cleaning_patient_closed.py

```diff
@@ -0,0 +1,46 @@
+from xml.etree import cElementTree as ElementTree
+
+from django.core.management.base import BaseCommand
+
+from casexml.apps.case.mock import CaseBlock
+from dimagi.utils.chunked import chunked
+
+from corehq.apps.es import CaseSearchES
+from corehq.apps.hqcase.utils import submit_case_blocks
+from corehq.apps.users.util import SYSTEM_USER_ID, username_to_user_id
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+
+BATCH_SIZE = 100
+DEVICE_ID = __name__ + ".update_closed"
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+        parser.add_argument('--username', type=str, default=None)
+
+    def handle(self, domain, **options):
+        if options["username"]:
+            user_id = username_to_user_id(options["username"])
+            if not user_id:
+                raise Exception("The username you entered is invalid")
+        else:
+            user_id = SYSTEM_USER_ID
+
+        case_ids = CaseSearchES().domain(domain).xpath_query(domain, "closed != ''").values_list("_id", flat=True)
```

Comment:
(non blocking) to get only the relevant cases should this be `closed = 'referral_hc_work3_entire_period'`?
You confirmed that those are all the cases as of now, but this just makes the intention clear to remove incorrectly set values.

Reply:
I think this doesn't matter either way since it's a one-off script, going to leave it as is.